packages/@aws-cdk/aws-apigateway/test/test.restapi.ts

```diff
@@ -586,23 +586,23 @@ export = {
       ResourceId: { Ref: "myapichildA0A65412" },
       Integration: { Type: 'AWS' },
       AuthorizerId: 'AUTHID',
-      AuthorizationType: 'COGNITO_USER_POOLS',
```

Comment:
Why was this test modified?

Reply:
[CF doc for Method](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-apigateway-method.html#aws-resource-apigateway-method-properties) states in the the `AuthorizationType` property section - "If you specify the AuthorizerId property, specify CUSTOM for this property. "

The original change contained validation to this effect and hence the test needed to be modified.

Since then, I think there is a documentation error here because the rule is not true when using cognito for authentication. In this case, both the `AuthenticationType` property on the `Method` and `type` property on the `Authorizer` should be `COGNITO_USER_POOL`.

I've reverted changes to this test file. At some point in the future, we might need additional validation, but I'd rather not mix this up with this (already long) PR.